docs/admin-invitations/0.1/README.md

```diff
@@ -13,151 +10,204 @@ a new DID to be provisioned for the invitation.
 - This version lacks paging of invitation lists.
 - This version lacks a way to filter invitations when listing.
 - The first example for each protocol message is for general usage; the second example specific to the ACA-Py Plugin Toolbox
-
+- Invitation create method will be changed slightly; the user will be able to specify whether to use Out-Of-Band or Connections protocol
 ### Protocol Messages
 - [create](#create)
+- [oob-create](#oob-create)
 - [invitation](#invitation)
+- [oob-invitation](#oob-invitation)
 - [invitation-get-list](#invitation-get-list)
 - [invitation-list](#invitation-list)
-
+Toolbox Plugin agents are able to send and receive either [Connections](https://github.com/hyperledger/aries-rfcs/blob/main/features/0160-connection-protocol/README.md#1-connection-request) Protocol or [Out-Of-Band](https://github.com/hyperledger/aries-rfcs/tree/main/features/0434-outofband) Protocol invitations, through the Admin Protocol messages. Agents receiving Connections Protocol invitations will use Connections Protocol to connect to the sender. Agents receiving OOB Protocol invitations will be able to choose between using Connections or [DID Exchange](https://github.com/hyperledger/aries-rfcs/tree/main/features/0023-did-exchange) protocols to respond and connect to the sender, which will be listed in the invitation's [handshake protocols](#out-of-band-specific-attributes). 
 ## Message Definitions
-
 > _**Note:**_ Some attributes are omitted from the examples in this section,
 > such as the `@id` attribute. The omission is for brevity and clarity of the
 > example and it should be assumed that each example message contains all
 > attributes required to be processed by the receiving agent.
-
 ### Common Attributes
 The following attributes are common across all messages in this protocol. Each time an attribute
  appears, it means the same thing.
-
 `alias`: The name of the invitation as viewed by the inviter.
-
 `label`: The label presented inside the invitation as a suggestion of what to name the connection.
-
 `group`: The name of the group that resulting connection(s) should receive.
-
 `auto_accept`: True if connections from this invitaiton should be automatically accepted.
-
 `multi_use`: True of this invitation is expected to be used multiple times.
-
+`type`: Within `connection.invitation`, specifies protocol of the invitation (Out-Of-Band or Connections). 
```

Comment:
The attribute is actually `invitation_type` and it is contained at the root of the messages that use it rather than within `connection.invitation`.